packages/ui-codemods/lib/updateV8Props.ts

```diff
@@ -22,26 +22,37 @@
  * SOFTWARE.
  */
 
+import { API, FileInfo } from 'jscodeshift'
+import formatSource from './utils/formatSource'
+
 /**
  * Renames components theme={} prop to themeOverride={}
```

Comment:
This is a codemod that we've written for v8, I've moved it to `ui-codemods` and converted it to TS